include/sparrow/arrow_interface/arrow_array_schema_info_utils.hpp

```diff
@@ -91,9 +94,46 @@ namespace sparrow
         SIZES_64BIT,
     };
 
+    constexpr std::array<buffer_type, 2> buffers_types_validity_data{buffer_type::VALIDITY, buffer_type::DATA};
+    constexpr std::array<buffer_type, 2> buffers_types_validity_offset32{
+        buffer_type::VALIDITY,
+        buffer_type::OFFSETS_32BIT
+    };
+    constexpr std::array<buffer_type, 2> buffers_types_validity_offset64{
+        buffer_type::VALIDITY,
+        buffer_type::OFFSETS_64BIT
+    };
+    constexpr std::array<buffer_type, 3> buffers_types_validity_offsets32_data{
+        buffer_type::VALIDITY,
+        buffer_type::OFFSETS_32BIT,
+        buffer_type::DATA
+    };
+    constexpr std::array<buffer_type, 3> buffers_types_validity_offsets64_data{
+        buffer_type::VALIDITY,
+        buffer_type::OFFSETS_64BIT,
+        buffer_type::DATA
+    };
+    constexpr std::array<buffer_type, 3> buffers_types_validity_offsets32_sizes32{
+        buffer_type::VALIDITY,
+        buffer_type::OFFSETS_32BIT,
+        buffer_type::SIZES_32BIT
+    };
+    constexpr std::array<buffer_type, 3> buffers_types_validity_offsets64_sizes64{
+        buffer_type::VALIDITY,
+        buffer_type::OFFSETS_64BIT,
+        buffer_type::SIZES_64BIT
+    };
+    constexpr std::array<buffer_type, 1> buffers_types_validity{buffer_type::VALIDITY};
+    constexpr std::array<buffer_type, 1> buffers_types_types_ids{buffer_type::TYPE_IDS};
+    constexpr std::array<buffer_type, 2> buffers_types_types_ids_offset32{
+        buffer_type::TYPE_IDS,
+        buffer_type::OFFSETS_32BIT
+    };
+    constexpr std::array<buffer_type, 0> buffers_types_empty{};
```

Comment:
I dont like this

Reply:
having this exhaustive amount of variables with long unreadable names  is hard to read and a code smell IMO

Reply:
Done 👍